swig/test_ChunkedArray_manually.cpp

```diff
@@ -0,0 +1,179 @@
+/**
+ * Tests for ChunkedArray.
```

Comment:
I think test file should go to `tests` folder. Although I'm not familiar with cpp tests, so might be very wrong. WDYT?

Reply:
Completely agree :)

Let me just see if the C++ gtest MR (#3555) is merged and I'll even migrate this to gtest.

Reply:
+1 would be best to put this in tests folder

Reply:
I made a `tests` folder inside `swig` for the time being, as I'm almost positive this would break the C++ tests that will be coming in MR #3555 as this is not gtest-compatible yet.

I can always open another PR to moved ChunkedArray outside of swig to some C++ helper module and add port the tests to gtest when that PR is complete.

Reply:
Thank you!